### PR TITLE
Added plugin flashforge

### PR DESCRIPTION
**Pull request by Mrnt**

### CODE REVIEW

File: _plugins/FlashForge.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  # Compatible Python version
+  #
+  # Plugins should aim for compatibility for Python 2 and 3 for now, in which case the value should be ">=2.7,<4".
+  #
+  # Plugins that only wish to support Python 3 should set it to ">=3,<4". 
+  #
+  # If your plugin only supports Python 2 (worst case, not recommended for newly developed plugins since Python 2
+  # is EOL), leave at ">=2.7,<3"
+      
+  python: ">=2.7,<3"
```
Comment (jneilliii) [inline _plugins/FlashForge.md:81]: @Mrnt your plugin's init.py file indicates that it is Python 3 compatible, but this flag indicates Python 2 only. Please correct.

Comment (Mrnt) [reply _plugins/FlashForge.md]: Oops, thanks for spotting thank @jneilliii !

**Comment by jneilliii** [comment]
Thanks, as soon as @kantlivelong or @foosel has a chance to review you should be good to go.

**Comment by kantlivelong** [comment]
@mrnt Nice work

**Comment by Mrnt** [comment]
@kantlivelong thanks!

**Pull request merged**